htdocs/settings_dialog.php

```diff
@@ -21,6 +22,55 @@
 <legend>Hostname regex</legend>
 <input type="input" name="hostfilter" value="<?php echo $_COOKIE['nagdash_hostfilter']; ?>">
 </fieldset>
+<legend>Last state change</legend>
+<div class="settings_group">
+<span class="settings_element">
+<select name="select_last_state_change" style="width:110px;">
+<?php
+foreach ($select_last_state_change_options as $time_in_seconds => $time_in_english) {
+    $selected_last_state_change_option = "";
+    if (isset($_COOKIE['select_last_state_change']) && $_COOKIE['select_last_state_change'] == $time_in_seconds) {
+        $selected_last_state_change_option = "selected";
+    }
+    echo "<option value=" . $time_in_seconds . " " . $selected_last_state_change_option . ">" . $time_in_english . "</option>";
+}
+?>
+</select>
+</span>
+<span class="settings_element"> ago</span>
+<legend>Sort options</legend>
+<?php
+// If the config option 'sort_by_time' is true, check if the user is overriding it.
+// If not, let the config option take effect.
+$checked_sort_by_time = "";
+$checked_sort_descending = "";
+if ($sort_by_time) {
+    if (isset($_COOKIE['sort_by_time']) && $_COOKIE['sort_by_time'] == '0') {
+        $checked_sort_by_time = "";
+    } else {
+        $checked_sort_by_time = "checked";
+    }
+} elseif (isset($_COOKIE['sort_by_time']) && $_COOKIE['sort_by_time'] == '1') {
+    $checked_sort_by_time = "checked";
+}
+// Does the user want to sort in descending order?
+if (isset($_COOKIE['sort_descending']) && $_COOKIE['sort_descending'] == '1') {
+        $checked_sort_descending = "checked";
+}
+echo '<span class="settings_element">';
+echo 'Sort by time?';
+echo '<label class="checkbox inline tag_label">';
+echo '<input type="checkbox" name="sort_by_time"' . $checked_sort_by_time . '>';
+echo '</label>';
+echo '</span>';
+echo '<span class="settings_element">';
+echo 'Descending?';
+echo '<label class="checkbox inline tag_label">';
+echo '<input type="checkbox" name="sort_descending"' . $checked_sort_descending . '>';
+echo '</label>';
+echo '</span>';
```

Comment:
can we move this block out of the `<?php` clause and make it plain HTML and only activate php for the variable stuff? Would make it much easier to read or change.